src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -60,8 +60,27 @@ public class ADDR_Contact_TDTM extends TDTM_Runnable {
         Map<Id,Account> mapAccountIdAccount = null;
         boolean acctHasAddress = false;
         boolean acctChanged = false;
+        boolean isBefore = (triggerAction == TDTM_Runnable.Action.BeforeUpdate || triggerAction == TDTM_Runnable.Action.BeforeInsert);
+
+        if (isBefore) {
```

Comment:
Thanks. There's an `if` online 84 that bail's out of address management is disabled, plus there's another `if (isBefore)` block starting at line 83. Wondering if this block needs to be merged into that block; and both are after the bail-out check.